cmd/titus-vpc-tool/setup_container.go

```diff
@@ -33,6 +33,7 @@ func setupContainercommand(ctx context.Context, v *pkgviper.Viper, iipGetter ins
 	}
 
 	cmd.Flags().Int("netns", 3, "The File Descriptor # of the network namespace to setup")
+	cmd.Flags().Int("network-mode", 0, "Type of networking to setup, uses the enum")
```

Comment:
:-/. Why not make this a string?

Reply:
I can do that.

Ooo yea looks like in the .pb.go it has all the mappers I want to make this sane. Great.